src/parsing.typ

```diff
@@ -64,15 +64,18 @@
   return result.replace(",", ".").replace(sym.minus, "-")
 }
 
-#let number-to-stringw(number) = {
+#let number-to-string-table(number) = {
   let result
   if type(number) == str { result = number }
   else if type(number) in (int, float) { result = str(number) }
-  else if type(number) == content  { result = content-to-stringw(number) } 
+  else if type(number) == content  { result = content-to-string-table(number) } 
   else { result = none }
   if result == none { return none }
   if type(result) != array { result = (result, none, none) }
   result.at(0) = result.at(0).replace(",", ".").replace(sym.minus, "-")
+  if result.len() == 0 or result.at(0).at(0) not in "0123456789+-.," { 
```

Comment:
If we've already replaced `,` with `.` in the line above, can we remove `,` from this valid char set?

Reply:
That's right, well observed.